lib/Rex/Commands/Iptables.pm

```diff
@@ -449,20 +503,28 @@ Remove all iptables rules.
 =cut
 
 sub iptables_clear {
+  my (@params) = @_;
+  my $ip_version = _get_ip_version( \@params );
 
   for my $table (qw/nat mangle filter/) {
-    iptables t => $table, F => '';
-    iptables t => $table, X => '';
+
+    # NAT for IPv6 supported by iptables >= v1.4.18
+    if ( $table eq "nat" && $ip_version == -6 ) {
+      next if _iptables_version() < 1_004_018;
```

Comment:
These three conditions might be unified intoa single `next if A && B && C`. What do you think?